spark_log_parser/parsing_models/application_model.py

```diff
@@ -194,20 +195,25 @@ def __init__(self, eventlogpath, bucket=None, stdoutpath=None, debug=False):  #
 
                 elif event_type == "SparkListenerEnvironmentUpdate":
 
-                    curKeys = json_data["Spark Properties"].keys()
+                    spark_properties = json_data["Spark Properties"]
+                    event_keys = spark_properties.keys()
```

Comment:
This just puts `json_data["Spark Properties"]` into a variable that we'll use elsewhere, and renames `curKeys` to follow the snake_case convention for variable names in Python.